pkgs/tools/filesystems/gcsfuse/default.nix

```diff
@@ -0,0 +1,23 @@
+# This file was generated by go2nix.
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
```

Comment:
Only `buildGoPackage` and `fetchgit` are used. Can you please remove the others?